clients/js/test/_common.ts

```diff
@@ -63,6 +64,18 @@ import {
 } from '../src/index.js';
 import { generateTreeOfSize } from './_merkle.js';
 
+export const COMPUTE_300K_IX = (() => {
+  return getSetComputeUnitLimitInstruction({
+    units: 300_000,
+  });
+})();
+
+export const COMPUTE_500K_IX = (() => {
+  return getSetComputeUnitLimitInstruction({
+    units: 500_000,
+  });
+})();
+
```

Comment:
Why do the unnamed closure?

Reply:
Just to make it a little cleaner to set compute units in tests:

```
    await pipe(
      await createDefaultTransaction(client, traderB),
      (tx) => appendTransactionMessageInstruction(COMPUTE_500K_IX, tx),
      (tx) => appendTransactionMessageInstruction(buyNftIx, tx),
      (tx) => signAndSendTransaction(client, tx)
    );
```